### PR TITLE
Fix part of #2003: Replacing space_128dp

### PR DESCRIPTION
**Pull request by Arjupta**
## Explanation

Fixes part of #2003
This PR replaces the use of space_128dp with dimensions of contextful names

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Arjupta** [comment]
@anandwana001 can you help me this failing test. I can't find the reason for it

`org.oppia.android.domain.oppialogger.loguploader.LogUploadWorkerTest > testWorker_logException_withoutNetwork_enqueueRequest_verifySuccess FAILED
    expected: SUCCEEDED
    but was : RUNNING
        at org.oppia.android.domain.oppialogger.loguploader.LogUploadWorkerTest.testWorker_logException_withoutNetwork_enqueueRequest_verifySuccess(LogUploadWorkerTest.kt:161)
`

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -90,4 +89,40 @@
   <dimen name="drag_drop_content_group_item_margin">8dp</dimen>
   <dimen name="walkthrough_final_no_button_margin_end">8dp</dimen>
   <dimen name="walkthrough_final_yes_button_margin_start">8dp</dimen>
+
+  <dimen name="content_item_question_view_margin_end">128dp</dimen>
```
Comment (anandwana001) [inline app/src/main/res/values/dimens.xml]: Looks like now this file is going to large, so can we arrange them in some groups like feature way?

Example 
```XML
<!-- Question -->
<!-- Exploration -->
<!-- Input interaction -->
<!-- Fraction interaction -->
<!-- Drag Drop interaction -->
...
...
...
```

@rt4914 WDYT?

File: app/src/main/res/layout-sw600dp-port/content_item.xml [hunk 2]
```diff
@@ -50,8 +50,8 @@
     app:questionSplitViewPaddingTop="@{@dimen/space_0dp}"
     app:questionViewMarginApplicable="@{!viewModel.hasConversationView &amp;&amp; !viewModel.isSplitView}"
     app:questionViewMarginBottom="@{@dimen/space_0dp}"
-    app:questionViewMarginEnd="@{@dimen/space_128dp}"
-    app:questionViewMarginStart="@{@dimen/space_128dp}"
+    app:questionViewMarginEnd="@{@dimen/content_item_question_view_margin_end}"
```
Comment (anandwana001) [inline app/src/main/res/layout-sw600dp-port/content_item.xml:53]: Does this file only used for a Question or is there any other place where we are using this file?

If using at multiple places, we can go for more generic names here like `content_item_margin_end `

File: app/src/main/res/layout-sw600dp-port/continue_interaction_item.xml [hunk 3]
```diff
@@ -46,8 +46,8 @@
     app:questionSplitViewPaddingTop="@{@dimen/space_0dp}"
     app:questionViewMarginApplicable="@{!viewModel.hasConversationView &amp;&amp; !viewModel.isSplitView}"
     app:questionViewMarginBottom="@{@dimen/space_0dp}"
-    app:questionViewMarginEnd="@{@dimen/space_128dp}"
-    app:questionViewMarginStart="@{@dimen/space_128dp}"
+    app:questionViewMarginEnd="@{@dimen/continue_interaction_item_question_view_margin_end}"
```
Comment (anandwana001) [inline app/src/main/res/layout-sw600dp-port/continue_interaction_item.xml:49]: same here `continue_interaction_item_margin_end `

Comment (anandwana001) [reply app/src/main/res/layout-sw600dp-port/continue_interaction_item.xml:49]: ditto else where

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @ARJUPTA 
Given the first look, added few thoughts, might need to finalize

@rt4914 added a few points

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (Arjupta) [reply app/src/main/res/values/dimens.xml]: Yes @anandwana001 I have thought to do this in the last PR I will be making on this issue. I will resolve all the conflicts and will group them also.

File: app/src/main/res/layout-sw600dp-port/content_item.xml (hunk 2 above)
Comment (Arjupta) [reply app/src/main/res/layout-sw600dp-port/content_item.xml:53]: In some files, we have two types of margin, one of them is exploration view and other is question view. So just to differentiate between those two I had to name these variables accordingly.

File: app/src/main/res/layout-sw600dp-port/continue_interaction_item.xml (hunk 3 above)
Comment (Arjupta) [reply app/src/main/res/layout-sw600dp-port/continue_interaction_item.xml:49]: Same comment as above

File: app/src/main/res/layout-sw600dp-port/content_item.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/layout-sw600dp-port/content_item.xml:53]: @anandwana001 @ARJUPTA I agree with Arjun. We should use very specific names here considering that the UI for items related to exploration/question are really complex.

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/values/dimens.xml]: @anandwana001 I agree. That would actually make this file more readable.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by rt4914** [comment]
@anandwana001 Will have to review this.

**Comment by anandwana001** [comment]
unassigning myself will take a look once all the checks pass.

**Comment by Arjupta** [comment]
@anandwana001 please review the changes you suggested

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @ARJUPTA

**Comment by Arjupta** [comment]
@rt4914 PTAL

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**